scripts/commit-msg-length-emojis.js

```diff
@@ -0,0 +1,48 @@
+const emojiLengths = {
+    '🗨️' : 3 ,
+    '🌳' : 2,
+    '🎀' : 2,
+    '🔣' : 2,
+    '🛠️' : 3,
+    '🧑‍🏫' : 5,
+    '💭' : 2,
+    '🎛️' : 3,
+    '🗺️' : 3,
+    '🫓' : 2,
+    '❄️' : 2,
+    '🛤️' : 3,
+    '🖌️' : 3,
+    '👼' : 2,
+    '🤕' : 2,
+    '🧰' : 2,
+    '📦' : 2,
+    '⌨️' : 2,
+    '📙' : 2,
+    '🧩' : 2,
+    '🎨' : 2,
+    '🔔' : 2,
+    '📂' : 2,
+    '🪨' : 2,
+    '🗒️' : 3,
+    '📇' : 2,
+    '👔' : 2,
+    '🐚' : 2,
+    '🔁' : 2,
+    '💾' : 2,
+    '📠' : 2,
+    '⚙️' : 2,
+    '👤' : 2,
+    '🖥️' : 3
+}
+
+function emojifiedLength (string){
+    let len = string.length;
+    for (const emoji in emojiLengths) {
+        const re = new RegExp(emoji,'g');
+        const count = (string.match(re) || []).length;
+        if (string.includes(emoji)) len -= count * (emojiLengths[emoji]-1);
+    };  
+    console.log(len)
```

Comment:
Looks a bit weird to have this function work by itself just via a `console.log`. Maybe just opt for a normal return here and put the `console.log` inside the `node -e` part of the git hook?